app/src/hooks/api/useUserApi.ts

```diff
@@ -19,6 +19,17 @@ const useUserApi = (axios: AxiosInstance) => {
     return data;
   };
 
+  /**
+   * Get user from userId
+   *
+   * @param {number} userId
+   * @return {*}  {Promise<IGetUserResponse>}
+   */
+  const getUserById = async (userId: number): Promise<IGetUserResponse> => {
+    const { data } = await axios.get(`/api/user/${userId}/get`, { params: { userId: userId } });
```

Comment:
You don't need the parms object.  The API will extract the userId value directly from the URL (because the url matches the path with the `{userId}` folder which it uses to recognize variables in the URL.

```suggestion
    const { data } = await axios.get(`/api/user/${userId}/get`);
```

(Also slightly confusing on Axios's part, but that params object adds items to the query string.  So what you had originally would actually produce: `/api/user/123/get?userId=123`)